.nvmrc

```diff
@@ -1 +1 @@
-v10.13.0
```

Comment:
I can't run tests with node version 10.13.0. I get the error `ReferenceError: describe is not defined`.

Reply:
fixed in 2f22fde. I'm now using `lts/*` which is the most recent lts version.